prog/vm/github_runner.rb

```diff
@@ -6,7 +6,7 @@ class Prog::Vm::GithubRunner < Prog::Base
   subject_is :github_runner
 
   def self.assemble(installation, repository_name:, label:, default_branch: nil)
-    unless Github.runner_labels[label]
+    unless Github.predefined_runner_labels[label] || GithubCustomLabel.where(label: label, installation_id: installation.id).any?
```

Comment:
If we keep `label` and `actual_label` as separate fields, we can continue using `label` throughout the codebase without having to check whether it comes from a predefined or a custom label each time.